cmd/cozystack-operator/main.go

```diff
@@ -0,0 +1,995 @@
+		os.Exit(1)
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
+	defer cancel()
+
+	setupLog.Info("Starting bootstrap phase 1: fluxcd installation")
+	if err := runBootstrapPhase1(ctx, bootstrapClient); err != nil {
+		setupLog.Error(err, "bootstrap phase 1 failed")
+		os.Exit(1)
+	}
+
+	// Wait for GitRepository CRD (needed for PlatformConfiguration controller)
+	// HelmRelease CRD is already waited for in phase 1
+	setupLog.Info("Waiting for GitRepository CRD (needed for PlatformConfiguration controller)")
+	if err := waitForCRDs(ctx, bootstrapClient, "gitrepositories.source.toolkit.fluxcd.io"); err != nil {
+		setupLog.Error(err, "failed to wait for GitRepository CRD, PlatformConfiguration controller will not be registered")
+	}
+
+	// Now that CRDs are available, we can start the controller manager
+	// The controller manager needs CRDs to be registered in the scheme
+	setupLog.Info("Starting controller manager (CRDs are now available)")
+	mgr, err := ctrl.NewManager(config, ctrl.Options{
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress:   metricsAddr,
+			SecureServing: secureMetrics,
+		},
+		WebhookServer: webhook.NewServer(webhook.Options{
+			Port: 9443,
+		}),
+		HealthProbeBindAddress: probeAddr,
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "platform-operator.cozystack.io",
+		// LeaderElectionReleaseOnCancel defines if the leader should step down voluntarily
+		// when the Manager ends. This requires the binary to immediately end when the
+		// Manager is stopped, otherwise, setting this significantly speeds up voluntary
+		// leader transitions as the new leader don't have to wait LeaseDuration time first.
+		//
+		// In the default scaffold provided, the program ends immediately after
+		// the manager stops, so would be fine to enable this option. However,
+		// if you are doing or is intended to do any operation such as perform cleanups
+		// after the manager stops then its usage might be unsafe.
+		// LeaderElectionReleaseOnCancel: true,
+	})
+	if err != nil {
+		setupLog.Error(err, "unable to start manager")
+		os.Exit(1)
+	}
+
+	// Setup PlatformReconciler
+	firstReconcileDone := make(chan struct{})
+	platformReconciler := &operator.PlatformReconciler{
+		Client:             mgr.GetClient(),
+		Scheme:             mgr.GetScheme(),
+		FirstReconcileDone: firstReconcileDone,
+	}
+	if err = platformReconciler.SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "Platform")
+		os.Exit(1)
+	}
+
+	bundleReconciler := &operator.CozystackBundleReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}
+	if err = bundleReconciler.SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "CozystackBundle")
+		os.Exit(1)
+	}
+
+	resourceDefinitionReconciler := &operator.CozystackResourceDefinitionReconciler{
```

Comment:
You already have a CozyRD reconciler in the Cozystack controller. You probably want to make changes in both places?

Reply:
All artefacts definition moved back to CozystackBundle